marconi/src/Marconi/CLI.hs

```diff
@@ -156,6 +158,10 @@ optionsParser =
                       <> Opt.help "disable epoch stakepool size indexers."
                       <> Opt.showDefault
                      )
+    <*> Opt.switch (Opt.long "disable-mintburn"
+                      <> Opt.help "disable mint/burn indexers."
+                      <> Opt.showDefault
```

Comment:
showDefault doesn't apply to switch and not required here. It has no effect!

BTW, I am at fault as I had erroneously, used them on a previous PR :(

Reply:
a4deee5dab1c664e71443373c7ceceb89e94228b